acceptance/configurations_test.go

```diff
@@ -94,9 +94,9 @@ var _ = Describe("Configurations", func() {
 			Expect(err).ToNot(HaveOccurred(), out)
 			Expect(out).To(MatchRegexp("Listing all configurations"))
 
-			Expect(out).To(MatchRegexp(fmt.Sprintf(`\| *%s *\| *%s *\| *%s *\|`, namespace1, configuration1, app1)))
-			Expect(out).To(MatchRegexp(fmt.Sprintf(`\| *%s *\| *%s *\| *\|`, namespace2, configuration1)))
-			Expect(out).To(MatchRegexp(fmt.Sprintf(`\| *%s *\| *%s *\| *\|`, namespace2, configuration2)))
+			Expect(out).To(MatchRegexp(fmt.Sprintf(`\| *%s *\| *%s *\|[^|]*\| *%s *\|`, namespace1, configuration1, app1)))
+			Expect(out).To(MatchRegexp(fmt.Sprintf(`\| *%s *\| *%s *\|[^|]*\| *\|`, namespace2, configuration1)))
+			Expect(out).To(MatchRegexp(fmt.Sprintf(`\| *%s *\| *%s *\|[^|]*\| *\|`, namespace2, configuration2)))
```

Comment:
In my next PR I'm going to add an helper for this regex. Meanwhile I've read that the `MatchRegexp` accepts `...args` as second argument, and it will use Sprintf under the hood. :slightly_smiling_face: 

https://onsi.github.io/gomega/#matchregexpregexp-string-args-interface

Reply:
Interesting. We can use that later, right ? Not for this PR ?